### PR TITLE
Update 'peers connected' UI to reflect active channels

### LINKED ISSUE #1973: Number of peers doesn't match
From Mike: this is because the number of peers is taken from webtorrent, and the peers in the table are taken from open channels.

Suggestion: get number of peers to just be a count of the rows in the table.

Screen Shot 2020-05-26 at 5 23 30 PM

### PR DESCRIPTION
**Pull request by snario**
Fixes #1973

### CODE REVIEW

File: packages/web3torrent/src/components/torrent-info/peer-network-stats/PeerNetworkStats.tsx [hunk 1]
```diff
@@ -7,14 +7,19 @@ export type PeerNetworkStatsProps = {torrent: TorrentUI};
...
+  ).length;
   return (
     !IdleStatuses.includes(torrent.status) && (
       <section className="PeerNetworkStats">
         {showTimeRemaining && `${torrent.parsedTimeRemaining}. `}
         {prettier(torrent.done || !torrent.downloadSpeed ? 0 : torrent.downloadSpeed)}
         /s down, {prettier(!torrent.uploadSpeed ? 0 : torrent.uploadSpeed)}/s up
         <br />
-        <strong data-test-selector="numPeers">{torrent.numPeers}</strong> Peers connected
+        <strong data-test-selector="numPeers">{numChannels}</strong> running channel
```
Comment (geoknee) [inline packages/web3torrent/src/components/torrent-info/peer-network-stats/PeerNetworkStats.tsx:21]:
```suggestion
        <strong data-test-selector="numPeers">{numChannels}</strong> running channels
```

**Comment by snario** [comment]
> What happens when a channel closes? Does numChannels decrement?

Yes

File: packages/web3torrent/src/components/torrent-info/peer-network-stats/PeerNetworkStats.tsx (hunk 1 above)
Comment (snario) [reply packages/web3torrent/src/components/torrent-info/peer-network-stats/PeerNetworkStats.tsx:21]: There is a statement after this checking for `=== 1` and add an `s` if false

Comment (geoknee) [reply packages/web3torrent/src/components/torrent-info/peer-network-stats/PeerNetworkStats.tsx:21]: Gotcha, sorry I missed it.

**Pull request merged**